frontendUiImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/GitMacheteErrorReportSubmitter.java

```diff
@@ -88,9 +88,7 @@ private URI constructNewGitHubIssueUri(IdeaLoggingEvent[] events, @Nullable Stri
   }
 
   private String getBugTemplate() throws IOException, AssertionError {
-    val classLoader = GitMacheteErrorHandler.class.getClassLoader();
-    assert classLoader != null;
-    return IOUtils.resourceToString("bug_report.md", StandardCharsets.UTF_8, classLoader);
+    return IOUtils.resourceToString("bug_report.md", StandardCharsets.UTF_8);
```

Comment:
we have another template (github issue template) under .github dir. I think there are some changes needed there

Reply:
Hmm actually I think it's okay that the other one (used for filling up issues manually) is slightly different from this one &mdash; do you have any specific changes in mind?

Reply:
Those logging groups, by module? I think i saw them there. Are theyv still applicable?

Reply:
Awww crap 🤦🏻 ofc, lemme update github issue template